shared/chat/conversation/list.desktop.js

```diff
@@ -114,11 +115,13 @@ class ConversationList extends Component<void, Props, State> {
 
   _invalidateChangedMessages (props: Props) {
     this.state.messages.forEach((item, index) => {
-      if (item.messageID !== props.messages.get(index, {}).messageID) {
+      const oldMessage = props.messages.get(index, {})
+
+      if (item.key !== oldMessage.key) {
```

Comment:
This seems like the correct semantics and fixes the underlying type issue (that timestamps don't have message ids).